pkg/network/tracer/tracer.go

```diff
@@ -216,6 +216,7 @@ func newTracer(config *config.Config) (*Tracer, error) {
 		if tr.processCache, err = newProcessCache(config.MaxProcessesTracked, defaultFilteredEnvs); err != nil {
 			return nil, fmt.Errorf("could not create process cache; %w", err)
 		}
+		go tr.processCache.RefreshTelemetry()
```

Comment:
I would put this inside newProcessCache.

Reply:
done